credentialProvider/src/main/java/org/frankframework/credentialprovider/delinea/DelineaCredentialFactory.java

```diff
@@ -69,10 +69,10 @@
  * }</pre>
  * </p>
  *
- * <p>Delinea secrets are referenced by <b>ID</b> in an authAlias, because they are retrieved from the Secret Server by id. See the <a
- * href="https://updates.thycotic.net/secretserver/restapiguide/TokenAuth/#tag/Secrets/operation/SecretsService_GetSecretV2">Get Secret</a> API.</p>
+ * <p>Delinea secrets are referenced by <b>ID</b> in an authAlias, because they are retrieved from the DelineaSecret Server by id. See the <a
+ * href="https://updates.thycotic.net/secretserver/restapiguide/TokenAuth/#tag/Secrets/operation/SecretsService_GetSecretV2">Get DelineaSecret</a> API.</p>
```

Comment:
```suggestion
 * <p>Delinea secrets are referenced by <b>ID</b> in an authAlias, because they are retrieved from the Delinea Secret Server by id. See the <a
 * href="https://updates.thycotic.net/secretserver/restapiguide/TokenAuth/#tag/Secrets/operation/SecretsService_GetSecretV2">Get Secret</a> API.</p>
```